src/lib/makeImportContext.ts

```diff
@@ -0,0 +1,384 @@
+// @ts-check
+// Copied and tweaked from @agoric/smart-wallet
```

Comment:
I suspect we can use without tweaking. If not we could give the SDK module some options for this use case.

Not worth slowing this down right now but consider putting an XXX tech debt comment to that effect.

Reply:
Done. I had trouble getting this to import, I think the other deps were giving it older versions of @agoric/vats and @agoric/store that broke, and I didn't want to fry the fish of upgrading all the dependencies yet to fix it.

Also, we would have had to patch to allow null slots like dapp-inter does.